src/js/components/award/shared/activity/ContractGrantActivityChart.jsx

```diff
@@ -130,10 +133,29 @@ const ContractGrantsActivityChart = ({
         // average difference
         const averageDifference = differences
             .reduce((acc, data) => acc + data, 0) / differences.length;
-        // subtracts the average difference from the first tick and updates the tick array
-        if (first) updatedTicks.splice(0, 0, updatedTicks[0] - averageDifference);
         // adds the average difference to the last tick and updates the tick array
         updatedTicks.push(updatedTicks[updatedTicks.length - 1] + averageDifference);
+        /**
+         * Since we are adding fake spacing to the top of the chart so the vertical line text
+         * never overlaps with the chart, and the lines will be descending in height to make
+         * sure the text never overlaps with other text. We must make sure the average difference
+         * is greater than the total height of the vertical line text. If it is not we will add
+         * another tick.
+         */
+        if (totalVerticalLineTextHeight > scale(averageDifference)) {
+            /**
+             * Now the question is how much space do we need to add? To determine this,
+             * we will get the difference from the total text height and then divide that
+             * by the average difference. If it is greater than 1, then we will add how ever
+             * many extra tick we need. :)
+             */
+            const difference = totalVerticalLineTextHeight - scale(averageDifference);
+            const howManyTicksToAdd = [];
+            for (let i = 0; i < Math.ceil(difference / averageDifference); i++) {
```

Comment:
FWIW Just figured out how to do this and allows us to disregard the whole for loop stuff:

```suggestion
[...new Array(Math.ceil(difference / averageDifference))]
  .forEach((val, i) => updatedTicks.push(updatedTicks[updatedTicks.length - 1] + averageDifference))
```